apps/minifront/src/components/header/menu/nav-style.tsx

```diff
@@ -0,0 +1,29 @@
+export const itemStyle = [
+  'font-bold text-[15px]',
+  'leading-10',
+  'text-muted-foreground',
+  'hover:text-muted',
+];
+
+export const triggerStyle = [
+  'px-4',
+  'w-full',
+  'font-bold text-[15px]',
+  'rounded-lg border border-light-brown',
+  'data-[state=open]:rounded-b-none',
+  'data-[state=open]:backdrop-blur-lg',
+  'data-[state=open]:bg-amber-950/[.10]',
+];
+
+export const dropdownStyle = [
+  'backdrop-blur-lg',
+  'bg-amber-950/[.10]',
+  'border-light-brown',
+  'rounded-b-lg border border-t-none',
+  'divide-y',
+  'divide-light-brown',
+  'transition',
+  'animate-expand-down',
+];
+
+export const linkStyle = ['cursor-pointer', 'mx-4'];
```

Comment:
[suggestion (ui, non-blocking)](https://conventionalcomments.org/): add `tracking-wide` tailwind class to increase letter spacing. Right now the letters feel too condensed